### PR TITLE
Interface changes

### PR DESCRIPTION
**Pull request by goneplaid**
Hey Steve, I was able to work on the site for a while last night and will get some more done tonight after dinner and a few errands. Let me know what you think of the navigation now.

I still need to clean up the posts a bit more as I removed some stuff that I thought was extraneous yet makes the posts look kind of bland now (and there are too many lines separating things in each post, which I'll edit as well).

### CODE REVIEW

**Comment by turboladen** [comment]
I'm checking the change out now... In the mean time, I do see that some of the view tests are failing though--would you mind fixing those up and checking them in?  `rake test` will run all of the tests for you.  There's one model test that I haven't been able to figure out why it's failing, so don't worry about that one; that failure looks like: 

```
Failure:
<5> expected but was
<4>.
test_get_posts_from_last_24_hours(PostTest)
test/unit/post_test.rb:170:in `block in <class:PostTest>'
```

File: app/assets/stylesheets/bootstrap_and_overrides.css.scss [hunk 1]
```diff
@@ -129,25 +125,21 @@ a:hover {
...
   top: -1px;
   left: -1px;
   padding: 3px 7px;
   font-size: 12px;
   font-weight: bold;
-  background-color:
-    #f5f5f5;
-  border: 1px solid
-  #ddd;
+  background-color: #f5f5f5;
```
Comment (turboladen) [inline app/assets/stylesheets/bootstrap_and_overrides.css.scss:141]: Do you think it's worth making this a sass variable?  I suppose only if it's reused elsewhere...?

File: app/assets/stylesheets/bootstrap_and_overrides.css.scss [hunk 2]
```diff
@@ -156,27 +148,20 @@ a:hover {
...
-    rgb(221, 221, 221);
-  border-left-color:
-    rgb(221, 221, 221);
-  color:
-    #9da0a4;
+  border-top-color: rgb(221, 221, 221);
+  border-right-color: rgb(221, 221, 221);
+  border-bottom-color: rgb(221, 221, 221);
+  border-left-color: rgb(221, 221, 221);
+  color: #9da0a4;
```
Comment (turboladen) [inline app/assets/stylesheets/bootstrap_and_overrides.css.scss:155]: Same question here--think any of these are worth turning into variables?

File: app/views/home/index.html.haml [hunk 3]
```diff
@@ -11,16 +11,27 @@
     = render partial: 'spokes/post', collection: @posts
 
 = content_for(:right_sidebar) do
-  .well.sidebar-nav{ style: "height: 100%;" }
-    %ul.nav.nav-list
-      %li.nav-header Spokes
+  -# .sidebar-nav{ style: "height: 100%;" } -- I took this out from .well.well-small below
```
Comment (turboladen) [inline app/views/home/index.html.haml:14]: If it doesn't seem to do anything, let's rip it out.  Can you remove?

Comment (goneplaid) [inline app/views/home/index.html.haml:14]: Hey Steve!

I'm looking into these issues now and will be working on resolving the
issues that came up. I'll go ahead and remove stuff that I've commented out
too; old habits die slowly :)

**Comment by turboladen** [comment]
I manually merged in _most_ of the changes here, @wilsonohyeah.  I left out the post changes on /home, as they didn't look quite finished.  Thanks a ton for these changes though--uber helpful!

**Pull request closed without merging**